rust/src/ssh/ssh.rs

```diff
@@ -64,7 +65,8 @@ pub enum SSHEvent {
 }
 
 #[repr(u8)]
-#[derive(Copy, Clone, PartialOrd, PartialEq, Eq)]
+#[derive(AppLayerState, Copy, Clone, PartialOrd, PartialEq, Eq)]
+#[suricata(alstate_strip_prefix = "SshState")]
```

Comment:
we're tracking this state per connection, not per direction, right? Does that make sense? The (firewall) rule engine expects us to traverse through each of these states per direction.

Reply:
No it is per direction